WordPressKitTests/Utilities/LoggingTests.swift

```diff
@@ -0,0 +1,70 @@
+import XCTest
+
+@testable import WordPressKit
+
+private class CaptureLogs: NSObject, WordPressLoggingDelegate {
+    var verboseLogs = [String]()
+    var debugLogs = [String]()
+    var infoLogs = [String]()
+    var warningLogs = [String]()
+    var errorLogs = [String]()
```

Comment:
Testing nitpick. To force all tests using `CaptureLogs` to use it as a `WordPressLoggingDelegate` implementation, we could make the probes `var`s `private(set)` so that they can only be read in the tests but not directly modified. The `private(set)` access, in my opinion, also makes it clearer that their purpose is to be used to check the object's state only.

```suggestion
    private(set) var verboseLogs = [String]()
    private(set) var debugLogs = [String]()
    private(set) var infoLogs = [String]()
    private(set) var warningLogs = [String]()
    private(set)var errorLogs = [String]()
```